frontend/src/Editor/Components/Button.jsx

```diff
@@ -3,15 +3,19 @@ import cx from 'classnames';
 var tinycolor = require('tinycolor2');
 
 export const Button = function Button(props) {
-  const { height, properties, styles, fireEvent, registerAction, id, dataCy } = props;
+  const { height, properties, styles, fireEvent, registerAction, id, dataCy, setExposedVariable } = props;
   const { backgroundColor, textColor, borderRadius, loaderColor, disabledState, borderColor } = styles;
 
   const [label, setLabel] = useState(properties.text);
   const [disable, setDisable] = useState(disabledState);
   const [visibility, setVisibility] = useState(styles.visibility);
   const [loading, setLoading] = useState(properties.loadingState);
 
-  useEffect(() => setLabel(properties.text), [properties.text]);
+  useEffect(() => {
+    setLabel(properties.text);
+    setExposedVariable('ButtonText', properties.text);
```

Comment:
```suggestion
    setExposedVariable('buttonText', properties.text);
```

All the exposed variables are with camelCase and we can maintain the same here